### PR TITLE
[MDB IGNORE] Allows credits to be claimed from recycling furnaces (plus access locking them)

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[content]

## What this does
enables this process
also gives wood, plastic and cardboard values for this, at half the value of metal.
adds an option on smelting consoles to lock them to an ID's access, or unlock all access if there's already some. this only applies to claiming credits.

## Why it's good
lets greytide make a tiny bit of extra cash on the side (not much though, as something with 50 sheets of metal worth of mats just gives 10 bucks back, with everything else new only giving back 5 for this amount)
allows the odd virtuous miner to let vox or space hobos or whatever claim them too

## How it was tested
toolboxes in it, turning on the admin debug variable showing how much value per material is available and checking them
inserting a janitor access ID, locking it with it, then inserting a blank id, then ejecting it, then unlocking it again with the janitor ID

## Changelog
:cl:
 * rscadd: Recycling furnaces can now have credits claimed from them for recycling, albeit meager amounts in the majority of cases, and with the tradeoff of deducting from materials smeltable for value claimed.
 * rscadd: Smelter consoles can now be ID locked or unlocked to prevent claiming of credits. Recycling ones start unlocked by default, mining ones default to mining access. Emagging also removes all access locks.

### CODE REVIEW

**Comment by wotisjanitor** [comment]
this should require a janitor ID, or else janitors will see all of their labors go to some greyshit

**Comment by RealestEstate** [comment]
Seconding the janitor ID proposal

**Comment by SECBATON-GRIFFON** [comment]
> this should require a janitor ID, or else janitors will see all of their labors go to some greyshit

added a way for janitors to lock it to themselves now

**Comment by wotisjanitor** [comment]
and if people dont like that i guess it could be hackable or something

**Comment by SECBATON-GRIFFON** [comment]
> and if people dont like that i guess it could be hackable or something

i was debating making emags override it yea

**Comment by DeityLink** [comment]
I'm sure there's a few cases where you can just loop a crafting recipe to perpetually recycle it generate cash from nothing. Not saying that economics matter but that's pretty dumb.

If disposals let you actually get rid of trash by shipping it out, and you didn't get both the materials AND the cash back, that'd make more sense to me.

**Comment by SECBATON-GRIFFON** [comment]
> I'm sure there's a few cases where you can just loop a crafting recipe to perpetually recycle it generate cash from nothing. Not saying that economics matter but that's pretty dumb.
> 
> If disposals let you actually get rid of trash by shipping it out, and you didn't get both the materials AND the cash back, that'd make more sense to me.

good stuff to consider, made it a tradeoff now

File: data/bunker.txt
Comment (west3436) [inline data/bunker.txt:1]: remove

**Review by west3436 — CHANGES_REQUESTED** [review]

File: data/bunker.txt
Comment (SECBATON-GRIFFON) [reply data/bunker.txt:1]: yea this thing got so annoying to remove after a while

Comment (SECBATON-GRIFFON) [reply data/bunker.txt:1]: yea this thing got so annoying to remove after a while. done

**Pull request merged**